src/scripts.rs

```diff
@@ -361,6 +361,59 @@ fn unvault_descriptor_csv<Pk: MiniscriptKey>(desc: &Descriptor<Pk>) -> u32 {
     }
 }
 
+fn unvault_descriptor_managers_threshold<Pk: MiniscriptKey>(
+    desc: &Descriptor<Pk>,
+    manager_keys: Vec<Pk>,
```

Comment:
No need for a Vec here, just take a slice and avoid cloning :)

Reply:
I'm not sure I can. I started with asking a slice of Pk but then I didn't know how to convert it to a slice of Pk::Hash without creating a vector... I'll try a bit again, if you have suggestions :)